guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,39 @@
+:_module-type: PROCEDURE
+
```

Comment:
```suggestion
```

I would delete the first two lines. If, at some point, we/you need metadata in each procedure, we should figure out a way to do this in one go.